src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -126,7 +123,15 @@
           </template>
         </li>
       </ul>
-    </v-row>
+    </v-row> -->
```

Comment:
I'm going to keep this around for a bit while figuring out what business data we need in the table.

Reply:
The lookup returns an object that contains some fields but not all. That's why you have to make other API calls to fetch the rest.

"A" companies are, I believe xpros. That means they're out-of-BC companies but we still have a COLIN/LEAR record for them, so we can handle them normally.

Foreign companies won't be returned by business lookup. Staff will enter those 3 data fields for them only. I would  consider their identified "special" in that it's user-entered and not necessarily unique. We will have to assume they are in good standing.

Yes, we have 2 flavours: LEAR (including A) + Foreign.